generators/createtodoapp/generator.mjs

```diff
@@ -33,6 +33,18 @@ export default class extends BaseGenerator {
 
         const props = await this.prompt(prompts, this.config);
 
+        // Make sure all derived properties are loaded
+        this.loadAppConfig(props);
+        this.loadDerivedAppConfig(props);
+        this.loadServerConfig(props);
+        this.loadDerivedServerConfig(props);
+        this.loadClientConfig(props);
+        this.loadDerivedClientConfig(props);
+        this.loadPlatformConfig(props);
+        this.loadTranslationConfig(props);
+
+        console.log({ props })
```

Comment:
should we remove this?

Reply:
Oops, the console.log wasn't meant to stay!

But the other calls should be kept